### PR TITLE
Export a function for network error handler

### PR DESCRIPTION
**Pull request by MichaelKim20**
I changed the code to make the error processing function export for use in Stoa.

### CODE REVIEW

File: src/modules/net/error/ErrorTypes.ts [hunk 1]
```diff
@@ -82,3 +82,47 @@ export class BadRequestError extends NetworkError
...
+
+/**
+ * It is a function that handles errors that occur during communication
+ * with a server for easy use.
+ * @param error This is why the error occurred
+ * @returns The instance of Error
+ */
+export function handleNetworkError (error: any): Error
+{
+    if (error.response && error.response.status && error.response.statusText)
```
Comment (TrustHenry) [inline src/modules/net/error/ErrorTypes.ts:94]: `error.response.statusText`
Isn't there an undefined case?

Comment (MichaelKim20) [reply src/modules/net/error/ErrorTypes.ts]:
```
    if (error.response && error.response.status && error.response.statusText)
```
Change the code above down to be accurate
```
    if (
            (error.response !== undefined) &&
            (error.response.status !== undefined) &&
            (error.response.statusText !== undefined)
    )
```

Comment (MichaelKim20) [reply src/modules/net/error/ErrorTypes.ts]: We checked it out in detail through the test code below.
If error.response.statusText is undefind, go to initial `else`.
```
else
{
    if (error.message !== undefined)
        return new Error(error.message);
    else
        return new Error("An unknown error has occurred.");
}
```
Test Code
```
    it ('Test for undefined', () =>
    {
        let check1 = (error: any): boolean =>  {
            if (error.response)
                return true;
            else
                return false;
        };

        let check2 = (error: any): boolean =>  {
            if (error.response !== undefined)
                return true;
            else
                return false;
        };

        assert.ok(!check1({}));
        assert.ok(!check1(!{response: ""}));
        assert.ok(check1({response: "0"}));
        assert.ok(check1({response: "1"}));
        assert.ok(check1({response: "false"}));
        assert.ok(check1({response: "true"}));
        assert.ok(!check1({response: 0}));
        assert.ok(check1({response: 1}));
        assert.ok(!check1({response: false}));
        assert.ok(check1({response: true}));
        assert.ok(check1({response: {}}));

        assert.ok(!check2({}));
        assert.ok(!check2(!{response: ""}));
        assert.ok(check2({response: "0"}));
        assert.ok(check2({response: "1"}));
        assert.ok(check2({response: "false"}));
        assert.ok(check2({response: "true"}));
        assert.ok(check2({response: 0}));
        assert.ok(check2({response: 1}));
        assert.ok(check2({response: false}));
        assert.ok(check2({response: true}));
        assert.ok(check2({response: {}}));
    });
```

**Comment by MichaelKim20** [comment]
@TrustHenry Do you have any other modifications or questions?

**Comment by MichaelKim20** [comment]
This has only changed the location of the source code that was already there and externalized. There is no problem, so I will merge it right away.

**Pull request merged**